schemas/bedrock-profile-http.js

```diff
@@ -159,6 +159,19 @@ const zcap = {
     referenceId: {
       title: 'Reference Id',
       type: 'string'
+    },
+    expires: {
+      title: 'W3C Date/Time',
+      description: 'A W3C-formatted date and time combination.',
+      type: 'string',
+      pattern: '^[1-9][0-9]{3}-(0[1-9]|1[0-2])-([0-2][0-9]|3[0-1])' +
+        'T([0-1][0-9]|2[0-3]):([0-5][0-9]):(([0-5][0-9])|60)(\\.[0-9]+)' +
+        '?(Z|((\\+|-)([0-1][0-9]|2[0-3]):([0-5][0-9])))?$',
```

Comment:
This pattern seems like it requires millisecond precision. Can we ensure that it's not required (so its optional and seconds can be sufficient)?

Reply:
millisecond was optional, but I guess it wasn't clear because '?' was on the next line. I moved it up to make it more clear.

Reply:
Oh, ok, that's good. Can you please add a test case that doesn't include it?

Reply:
Added a test.